### PR TITLE
[Feature] Add automatic OS detection and default script selection

### PR DESCRIPTION
**Pull request by sin-diesel**

### CODE REVIEW

File: scripts/install.sh [hunk 1]
```diff
@@ -3,12 +3,28 @@
 home_dir="gnu-koryavov"
 scipts_dir="scripts"
 
+ubuntu_default_editor="atril"
```
Comment (nerett) [inline scripts/install.sh:6]: It seems default document viewer usually depends on the de installed. You can use something like

```shell
if [[ $DESKTOP_SESSION == "plasma" ]]; then
    sed -i '/pdfviewer_script/s/=.*\.sh/=okular.sh/' config.conf
fi
```

(and I'm not sure Atril is default ubuntu document viewer)

Comment (sin-diesel) [reply scripts/install.sh:6]: Maybe not default, but it can surely be installed on Ubuntu. DE check makes sense

Comment (sin-diesel) [reply scripts/install.sh:6]: How widely is DESKTOP_SESSION environmental variable is supported? It seems like some of the distributions may not support it.

 https://unix.stackexchange.com/questions/116539/how-to-detect-the-desktop-environment-in-a-bash-script

Comment (nerett) [reply scripts/install.sh:6]: Yeah, this variable is not very widespread, but it looks like there is no much trouble with it on KDE/Gnome, and that's why it's the simpliest way to establish wether there is Okular/Evince/etc already installed (Kubuntu and Ubuntu use Okular by default by the way)

Comment (nerett) [reply scripts/install.sh:6]: But detecting document viewers by running `documentviewer --version` is better obviously (but it's not as easy due to the anti-unification)

Comment (sin-diesel) [reply scripts/install.sh:6]: I think for now we'll leave detection of os as it is, and switch to okular as default editor.

**Review by nerett — APPROVED** [review]

**Pull request merged**